solutions/instances/variables.tf

```diff
@@ -88,8 +221,8 @@ variable "log_analysis_tags" {
 
 variable "log_analysis_enable_archive" {
   type        = bool
-  description = "Whether to enable archiving on Log Analysis instances."
-  default     = true
+  description = "Whether to enable archiving on Log Analysis instances. If set to true, `log_analysis_provision` must also be set to true."
+  default     = false
```

Comment:
I think keep this default value to `true` since it only works if `log_analysis_provision` is true (and that is now defaulting to false)